### ISSUE TITLE
@keyframes "from"/"to" is detected as tag name

### ISSUE THREAD

**Issue by jens-duttke** [Moxio/stylelint-selector-tag-no-without-class#5]
In this code
```
@keyframes fade-in {
	from {
		transform: translateY(-40px);

		opacity: 0;
	}

	to {
		transform: translateY(0);

		opacity: 1;
	}
}
```
the "from" and the "to" keywords are wrongly reported as "Unexpected tag from without class qualifier".

My config looks like this:
```
'plugin/selector-tag-no-without-class': ['/./'],
```
because I want to prevent any tag-without-class usage in CSS Modules.

**Comment by aboks** [comment]
That's an interesting issue! Thanks for reporting, we will have a look at it.

**Comment by aboks** [comment]
Should be fixed in version 2.0.4.
